spec/abstract-jobs.html

```diff
@@ -1,6 +1,63 @@
 <emu-clause id="sec-abstract-jobs">
   <h1> Abstract Jobs </h1>
 
+  <emu-clause id="sec-weakref-processing-model">
+    <h1>Processing model of WeakRefs and FinalizationGroups</h1>
+
+    <p>
+      This specification does not make any guarantees about garbage collection.
+      Objects which are unreachable from ECMAScript may be released after long periods
+      of time, or never at all. For this reason, this specification uses the term
+      "may" when describing behavior triggered by garbage collection.
+    </p>
+
+    <p>
+      At any given time, if all references to a ECMAScript object are from a
+      [[Target]] field or internal slot, inlcuding the agent's [[KeptAlive]] field,
+      the implementation may replace the reference to the object in all such
+      [[Target]] fields or internal slots with the value ~empty~.
```

Comment:
What is "~empty~" ?

Reply:
It's a sentinel used in this specification inside WeakRefs when the GC nulls out a pointer.